esp/xslt/esdl2monitor.xslt

```diff
@@ -182,22 +183,26 @@ END;
   string csndServer := '127.0.0.1' : stored('cassandraServer', FORMAT(SEQUENCE(1)));
   string csndUser := '' : stored('cassandraUser', FORMAT(SEQUENCE(2)));
   string csndPassword := '' : stored('cassandraPassword', FORMAT(PASSWORD, SEQUENCE(3)));
-  string csndKeySpace := 'eclresultmonitoring' : stored('cassandraKeyspace', FORMAT(SEQUENCE(4)));
+
+  string csndKeySpaceFrom := 'monitors_a' : stored('fromKeyspace', FORMAT(SEQUENCE(4)));
+  string csndKeySpaceTo := 'monitors_a' : stored('toKeyspace', FORMAT(SEQUENCE(4)));
 
   string monAction := 'Create' : STORED('MonAction', FORMAT(SELECT('Create,Run'), SEQUENCE(5)));
   string userId := '' : stored('UserId', FORMAT(SEQUENCE(6)));
   string service_url := '' : stored('QueryURL', FORMAT(SEQUENCE(7)));
   string service_name := '' : stored('QueryName', FORMAT(SEQUENCE(8)));
-  string MonitorIdIn := '' : stored('MonitorId', FORMAT(SEQUENCE(9)));
 
+<xsl:if test="$diffaction='Run'">
```

Comment:
I see the default value is 'Run' but is it possible this param will contain other casing? RUN or run or rUN, etc?